client/src/androidTest/java/io/getstream/chat/android/client/poc/ClientInstrumentationTests.kt

```diff
@@ -70,14 +71,11 @@ class ClientInstrumentationTests {
         runOnUi {
             val client = ChatClient.Builder(apiKey, context).build()
             client.setUser(User(userId), token)
-            client.events()
-                .filter(ConnectedEvent::class.java)
-                .first()
-                .subscribe {
-                    client.events().subscribe { event ->
-                        connectedEventConsumer.onEvent(event)
-                    }
+            client.subscribeForSingle<ConnectedEvent> {
+                client.subscribe { event ->
```

Comment:
You can use function reference here:
```
client.subscribe(connectedEventConsumer::onEvent)
```

Also applies to other places.

Reply:
Nice catch, thanks!